app/src/main/java/com/github/fribourgsdp/radio/UserProfileActivity.kt

```diff
@@ -162,6 +155,9 @@ class UserProfileActivity : AppCompatActivity() {
                     user.save(this)
                 }
                 usernameField.setText(user.name)
+                googleSignInButton.setText("Sign out")
```

Comment:
Also here, maybe not use hardcoded strings.

Reply:
Done